spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
Yes, all understood. My assumption was that an optional `raw` would allow producers to omit the property when it is directly implied by value (i.e., a value for which `toString` is idempotent and outputs no line terminators or potential delimiters), reducing data duplication. Such looseness could easily be prohibited, though—even on a case-by-case basis in consumers, like @RReverser's suggestion that absence of `raw` implies absence of directives.